Updating kinesis log SAM app version

himsharma01 opened the pull request.

himanshu219 commented:
don't we need to upgrade these
https://github.com/SumoLogic/terraform-sumologic-sumo-logic-integrations/blob/master/aws/elb/elb.tf#L142
https://github.com/SumoLogic/terraform-sumologic-sumo-logic-integrations/blob/323ce6eb965e4c3f5c39a3f3aa07e46d9ace250b/aws/elasticloadbalancing/variables.tf#L114
https://github.com/SumoLogic/terraform-sumologic-sumo-logic-integrations/blob/323ce6eb965e4c3f5c39a3f3aa07e46d9ace250b/aws/cloudwatchlogsforwarder/cloudwatchlogsforwarder.tf#L193

Lets not create separate PR for each of the modules.
Also lets search for node12 version throughout the repository
and also the semantic version 1.0.5  throughout the repository

himsharma01 commented:
> don't we need to upgrade these https://github.com/SumoLogic/terraform-sumologic-sumo-logic-integrations/blob/master/aws/elb/elb.tf#L142 - This uses a different SAM app which is not updated.
> 
> https://github.com/SumoLogic/terraform-sumologic-sumo-logic-integrations/blob/323ce6eb965e4c3f5c39a3f3aa07e46d9ace250b/aws/elasticloadbalancing/variables.tf#L114
> 
> 
> https://github.com/SumoLogic/terraform-sumologic-sumo-logic-integrations/blob/323ce6eb965e4c3f5c39a3f3aa07e46d9ace250b/aws/cloudwatchlogsforwarder/cloudwatchlogsforwarder.tf#L193
> 
> Lets not create separate PR for each of the modules.
> 
> 1. Also lets search for node12 version in all the repositories - Checked no other occurrence of node12
> 2. and also the semantic version 1.0.5  in all the repositories - Done and updated PR
> 3. and search for loggroup connector modules's or serverless application's reference in all the repositories - Done and updated PR
